pkg/core/native/native_test/neo_test.go

```diff
@@ -331,6 +331,16 @@ func TestNEO_RecursiveGASMint(t *testing.T) {
 	neoValidatorInvoker.Invoke(t, true, "transfer", e.Validator.ScriptHash(), c.Hash, int64(1), nil)
 }
 
+func TestNEO_GetCommitteeAddress(t *testing.T) {
+	neoValidatorInvoker := newNeoValidatorsClient(t)
+	e := neoValidatorInvoker.Executor
+	committeeAddress, err := e.Chain.GetCommitteeAddress()
```

Comment:
It's stand-by, just take the committee from config, sort the keys and create the script manually to ensure it matches the expectations. We don't need to expose APIs via blockchain only for tests.